policyengine_us/variables/gov/states/mt/tax/income/deductions/itemized/mt_itemized_deductions.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_us.model_api import *
+
+
+class mt_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana itemized deductions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf"
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
```

Comment:
reading through the [legislation](https://leg.mt.gov/bills/mca/title_0150/chapter_0300/part_0210/section_0200/0150-0300-0210-0200.html) it seems like there is cap which is the standard deduction amount.